hw/ip/prim/util/generate_prim_mubi.py

```diff
@@ -23,14 +25,22 @@
 
 def main():
 
+    this_dir = os.path.dirname(__file__)
+    repo_top = os.path.normpath(os.path.join(this_dir, '../../../..'))
```

Comment:
I recommend:
- using `pathlib`.
- making `REPO_TOP` a global, as a lot of other scripts do.

Reply:
Yeah, I could switch to using pathlib. Honestly, I think it sometimes makes things more messy (and you don't end up with such nice relative paths :-D). I'll push up a version doing that.

About the globals: could do, but it means more _SHOUTING_ABOUT_THINGS which I'm not a big fan of. It's not like we're running `main` lots of times, after all :-)

Reply:
btw, for determining repo top, should we use something like `git rev-parse --show-toplevel`? That way if we ever move the script around it seems like things will just stay stable.